config/util/representation-conversion/default.json

```diff
@@ -27,16 +27,21 @@
             { "@type": "ErrorToQuadConverter" },
             {
               "@type": "ErrorToTemplateConverter",
-              "engine": { "@type": "HandlebarsTemplateEngine" },
-              "templatePath": "$PACKAGE_ROOT/templates/error/main.md",
-              "descriptions": "$PACKAGE_ROOT/templates/error/descriptions/",
+              "engine": {
+                "@type": "HandlebarsTemplateEngine",
+                "defaultTemplate_templateFile": "$PACKAGE_ROOT/templates/error/main.md"
+              },
+              "templatePath": "$PACKAGE_ROOT/templates/error/descriptions/",
               "contentType": "text/markdown",
```

Comment:
@joachimvh Maybe we need an `internal/fragment+html`, since `text/markdown` is actually cheating a bit.

Reply:
Never mind.